src/main/java/ch/njol/skript/expressions/ExprTarget.java

```diff
@@ -44,16 +34,31 @@
 import ch.njol.skript.registrations.Classes;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Mob;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityTargetEvent;
+import org.bukkit.util.Vector;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.List;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Target")
-@Description("For players this is the entity at the crosshair, while for mobs and experience orbs it represents the entity they are attacking/following (if any).")
-@Examples({"on entity target:",
-			"\tentity's target is a player",
-			"\tsend \"You're being followed by an %entity%!\" to target of entity"})
-@Since("<i>unknown</i> (before 2.1)")
+@Description("For players this is the entity at the crosshair, while for mobs and experience orbs it represents the" +
+	" entity they are attacking/following (if any).")
+@Examples({
+	"on entity target:",
+		"\tif entity's target is a player:",
+			"\t\tsend \"You're being followed by an %entity%!\" to target of entity",
+	"",
+	"reset target of entity # Makes the entity target-less",
+	"delete targeted entity of player # for players it will delete the target",
+	"delete target of last spawned zombie # for entities it will make them target-less"
+})
+@Since("<i>unknown</i> (before 2.1), INSERT VERSION (Reset)")
```

Comment:
Version ExprTarget was added was 1.4.2. Might aswell update this now.
```suggestion
@Since("1.4.2, INSERT VERSION (Reset)")
```